src/content/docs/guides/docker.md

```diff
@@ -7,10 +10,10 @@ instead** 🐳.
 <img src="https://img.shields.io/docker/pulls/lycheeverse/lychee?color=%23099cec&logo=Docker" alt="Docker Pulls Badge" />
 </a>
 
-We provide a handy official image over at [lycheeverse/lychee][docker-image],
```

Comment:
Does that mean link labels are not supported by Starlight?

Reply:
Oh they are, I can put it back how it was, if that is easier for you? I removed the link labels because I like to keep to DRY (don't repeat yourself) code, but that's such a small thing.